### DISCUSSION TITLE
Running install script from github release

### DISCUSSION THREAD

**Discussion by dudicoco** [jdx/mise#3431] [Q&A]
It seems that the mise install script is rendered and packed for release here: https://github.com/jdx/mise/blob/2cc97ca317df356da19bc9b25fb37cc74d89b8a4/scripts/release.sh#L68

However, I don't see `install.sh` within the github releases assets.

I would like to run a "locked" version of the install script.

**Comment by jdx** [comment]
you can just do this: `curl https://mise.run > install.sh`

**Reply by dudicoco** [reply]
I prefer to use the remote version directly but from a specific commit/release. 
That way I can have 100% reproducible installs both locally and in CI.

**Comment by Marukome0743** [comment]
This is now possible using the versioned `install.sh` attached to each mise GitHub release.

For example, the installer for mise v2026.8.0 can be downloaded from:

https://github.com/jdx/mise/releases/download/v2026.8.0/install.sh

I verified the actual release asset. Unlike `https://mise.run`, this script is tied to v2026.8.0: it contains the fixed version and SHA-256 checksums for that release’s artifacts. Signature files (`install.sh.sig` and `install.sh.minisig`) are also included in the release assets.

This allows the installer to be downloaded from a specific release URL and committed or cached for reproducible installations, without depending on the moving `mise.run` endpoint.

Release-specific installer scripts have been available since mise v2024.12.18.

*AI-assisted — Tool: Codex; model: OpenAI/GPT-5; version: unavailable.*
